cmd/bedFormat/bedFormat.go

```diff
@@ -40,11 +49,19 @@ func usage() {
 	flag.PrintDefaults()
 }
 
+type Settings struct {
+	InFile         string
+	OutFile        string
+	UCSCToEnsembl  bool
+	EnsemblToUCSC  bool
+	ScaleNameFloat float64
+}
+
 func main() {
 	var expectedNumArgs int = 2
 	var ensemblToUCSC *bool = flag.Bool("ensemblToUCSC", false, "Changes chromosome format type.")
 	var UCSCToEnsembl *bool = flag.Bool("UCSCToEnsembl", false, "Changes chromosome format type.")
-	var fields *int = flag.Int("fields", 4, "Specify the number of bed fields.")
+	var scaleNameFloat *float64 = flag.Float64("scaleNameFloat", math.MaxFloat64, "If float values are held in the name field, scale those values by this constant multiplier.")
```

Comment:
Should this scale value default to 1?

Reply:
We need an option to not scaleNameFloat for beds where non-float64 data is stored in the name field